### DISCUSSION TITLE
Is there a way to schedule email send?

### DISCUSSION THREAD

**Discussion by EdoardoCroci** [PHPMailer/PHPMailer#2884] [Q&A]
Hi, I would like to know if there is a way to schedule email send since i could find nothing from the docs.

**Comment by Synchro** [comment]
That's not up to PHPMailer; it's a higher-level application concern. You would delay the requirement to send something in your a app (perhaps storing a desired send time in your database), then trigger a send when you want it to happen via some scheduled task such as cron, queued jobs, etc.
